tests/appveyor/mssql.appveyor.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/4.8/phpunit.xsd"
+         backupGlobals="false"
+         colors="true"
+         bootstrap="../../vendor/autoload.php"
+>
+    <php>
+        <ini name="error_reporting" value="-1" />
+
+        <var name="db_type" value="pdo_sqlvr"/>
```

Comment:
Can you change the names of the config files to the convention `<dbal_driver_name>.appveyor.xml` so that we can have a test config for each driver? Then also please can you add another build for `sqlsrv` driver?